<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>501</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{background-color: rgb(9, 4, 36);}

</string>
    </property>
    <widget class="QPushButton" name="a">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>375</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: white;
border: 2px solid rgb(0, 85, 255);
border-left-color:rgb(0, 28, 85);
border-top-color:rgb(0, 28, 85);
border-radius: 20px ;

}

:hover{
background: rgb(0, 85, 255);
border-color:rgb(0, 85, 255);
}

:pressed{
background-color: rgb(0, 0, 70);
border-right-color:rgb(0, 0, 70);
border-bottom-color:rgb(0, 0, 70);
border-left-color:rgb(0, 0, 70);
border-top-color:rgb(0, 0, 70);
}





</string>
     </property>
     <property name="text">
      <string>Castlevania: Symphony of the Night</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="question_lbl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>1000</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt 'MS Shell Dlg 2';
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Question</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="b">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>300</y>
       <width>375</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: white;
border: 2px solid rgb(0, 85, 255);
border-right-color:rgb(0, 28, 85);
border-top-color:rgb(0, 28, 85);
border-radius: 20px ;

}

:hover{
background: rgb(0, 85, 255);
border-color:rgb(0, 85, 255);
}

:pressed{
background-color: rgb(0, 0, 70);
border-right-color:rgb(0, 0, 70);
border-bottom-color:rgb(0, 0, 70);
border-left-color:rgb(0, 0, 70);
border-top-color:rgb(0, 0, 70);
}




</string>
     </property>
     <property name="text">
      <string>Anhydrous Sodium Carbonate</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="d">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>370</y>
       <width>375</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: white;
border: 2px solid rgb(0, 85, 255);
border-right-color:rgb(0, 28, 85);
border-bottom-color:rgb(0, 28, 85);
border-radius: 20px ;

}

:hover{
background: rgb(0, 85, 255);
border-color:rgb(0, 85, 255);
}

:pressed{
background-color: rgb(0, 0, 70);
border-right-color:rgb(0, 0, 70);
border-bottom-color:rgb(0, 0, 70);
border-left-color:rgb(0, 0, 70);
border-top-color:rgb(0, 0, 70);
}





</string>
     </property>
     <property name="text">
      <string>Albus Valencium Horatio Kul Du</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="c">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>375</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
color: white;
border: 2px solid rgb(0, 85, 255);
border-left-color:rgb(0, 28, 85);
border-bottom-color:rgb(0, 28, 85);
border-radius: 20px ;

}

:hover{
background: rgb(0, 85, 255);
border-color:rgb(0, 85, 255);
}

:pressed{
background-color: rgb(0, 0, 70);
border-right-color:rgb(0, 0, 70);
border-bottom-color:rgb(0, 0, 70);
border-left-color:rgb(0, 0, 70);
border-top-color:rgb(0, 0, 70);
}





</string>
     </property>
     <property name="text">
      <string>3.14</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="diff_lbl">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>460</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
font: 15pt 'Courier';
color: white;
background:rgb(0, 85, 255);
border-radius: 15
}




</string>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="message">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>500</y>
       <width>680</width>
       <height>400</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">*#message{border: 4px solid rgb(0, 85, 255);
color:white;
background: rgb(9, 4, 36);
font: 16pt &quot;Modern&quot;;
}</string>
     </property>
     <property name="title">
      <string>Wrong</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="msg_lbl">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>341</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
background: rgb(9, 4, 36);
font: 18pt &quot;Modern&quot;;</string>
      </property>
      <property name="text">
       <string>You failed that question but the Guru is kind so he gives you another chance</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="retry_btn">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
font: 19pt 'Garamond';
color: white;
background-color: rgb(0, 85, 255);
border-radius: 30 ;
}

</string>
      </property>
      <property name="text">
       <string>Ok Thanks</string>
      </property>
     </widget>
     <widget class="QLabel" name="wizard">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>181</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../assets/wizard.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="story">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>25</y>
      <width>850</width>
      <height>450</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*#story{border: 2px solid rgb(0, 85, 255);
color:white;
background: rgb(9, 4, 36);
font: 16pt &quot;Modern&quot;;
border-radius: 30px}</string>
    </property>
    <property name="title">
     <string>Prologue </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="msg_lbl_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>770</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background: rgb(9, 4, 36);
font: 18pt &quot;Modern&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Very Well&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
